cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -460,3 +458,13 @@ def _add_dna_cases_to_dna_sample(
                 rna_dna_sample_case_map[rna_sample.internal_id][dna_sample.name].append(
                     case_object.internal_id
                 )
+
+    @staticmethod
+    def _filter_mip_dna_balsamic(subject_id_samples: Query) -> List[Any]:
```

Comment:
Make the method name more clear. Also, when we are trying to get specific data name it `def _get_bla_bla`. You can also have another look at the return type - the Query will return `models.Sample` when called